apps/web/src/contexts/modal/ModalContext.tsx

```diff
@@ -45,6 +45,7 @@ type ShowModalFnProps = {
   hideClose?: boolean;
   isPaddingDisabled?: boolean;
   onClose?: () => void;
+  onBypassClose?: () => void;
```

Comment:
could we add a comment above this line explaining what this prop does?

e.g. `// if provided, this callback will trigger as opposed to closing the modal`